extensions/mssql/src/reactviews/pages/TableExplorer/TableDataGrid.css

```diff
@@ -215,3 +215,45 @@
     color: var(--vscode-foreground);
     padding-left: 8px;
 }
+
+/* Deleted row styles */
+#tableExplorerGrid .slick-row.deleted-row {
+    background-color: #ffb3d9 !important; /* Light pink for light theme */
+}
+
+#tableExplorerGrid .slick-row.deleted-row:hover {
+    background-color: #ff99cc !important; /* Darker pink on hover */
+}
+
+#tableExplorerGrid .slick-row.deleted-row.selected {
+    background-color: #ff80bf !important; /* Even darker pink when selected */
+}
+
+#tableExplorerGrid .slick-row.deleted-row .slick-cell {
+    text-decoration: line-through;
+    color: #666 !important;
+    opacity: 0.7;
+}
+
+/* Dark mode deleted row styles */
+#tableExplorerGrid.dark-mode .slick-row.deleted-row,
+.table-explorer-grid-container.dark-mode #tableExplorerGrid .slick-row.deleted-row {
+    background-color: #4d1f3a !important; /* Dark pink for dark theme */
```

Comment:
Instead of hardcoding these, maybe we can use some fluent or vscode theme colors?

https://storybooks.fluentui.dev/react/?path=/docs/theme-colors--docs

Looks like many foreground and background colors tend to come in light, medium, and dark variants, plus then you have guaranteed contrasts for light and dark themes and between foreground and background.

Take a look at the `colorStatusDanger` family of colors.  Looks like there are foreground/background and hove/select state colors all already picked out, and they'd adjust automatically for light/dark mode.

Or maybe the vscode theme colors `var(--vscode-errorForeground)` and the like?